src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1791,7 +1792,7 @@ else if (clickedBlock != null && instance.config_claims_preventTheft && clickedB
             }
         }
 
-        //apply rule for note blocks and repeaters and daylight sensors //RoboMWM: Include flower pots
+        //apply rule for note blocks and repeaters and daylight sensors //RoboMWM: Include flower pots //PR: Include redstone ore
```

Comment:
The comment here identifying a behavior change from Robo is from before TechFortress was the official maintainer of GP iirc - would be better to rephrase the comment than keep adding junk.

Reply:
Makes sense, however similar to pressure plates, it still makes sense to protect it from players. For example. a redstone ore that is in the wall can only be activated by players, and could be used as a trigger for redstone doors, etc.